paasta_tools/marathon_tools.py

```diff
@@ -68,11 +69,21 @@
 log = logging.getLogger(__name__)
 logging.getLogger('marathon').setLevel(logging.WARNING)
 
+MarathonServers = namedtuple('MarathonServers', ['current', 'previous'])
+MarathonClients = namedtuple('MarathonClients', ['current', 'previous'])
```

Comment:
namedtuple is convenient for now, but actually I think we're going to want to define a class for this, since we'll want to define methods like `get_current_shard()` and `get_previous_shards()`

Reply:
but aren't those functions of a an instance?
I've already defined those functions in the MarathonInstanceConfig thingy

Reply:
Hm, yeah, that does seem like a better place to put that code.